pennylane/plugins/default_qubit.py

```diff
@@ -183,6 +187,51 @@ def Rot3(a, b, c):
     return Rotz(c) @ (Roty(b) @ Rotz(a))
 
 
+def CRotx(theta):
+    r"""Two-qubit controlled rotation about the x axis.
+
+    Args:
+        theta (float): rotation angle
+    Returns:
+        array: unitary 4x4 rotation matrix `
```

Comment:
```suggestion
        array: unitary 4x4 rotation matrix
```

Reply:
worthwhile to state the matrix here? e.g., could do
```
:math:`|0\rangle\langle 0|\otimes \mathbb{1} + |1\rangle\langle 1|\otimes \mathbb{1} R_x`
```
(also applies to other similar functions)